Fix build with recent argp-standalone.
I'll try and get this upstreamed so we don't have to carry it.

diff -Nur xen-4.14.2.orig/tools/configure.ac xen-4.14.2/tools/configure.ac
--- xen-4.14.2.orig/tools/configure.ac	2020-12-17 16:47:25.000000000 +0000
+++ xen-4.14.2/tools/configure.ac	2021-02-20 19:44:33.618002472 +0000
@@ -426,7 +426,7 @@
 AC_CHECK_LIB([iconv], [libiconv_open], [libiconv="y"], [libiconv="n"])
 AC_SUBST(libiconv)
 AC_CHECK_HEADER([argp.h], [
-AC_CHECK_LIB([argp], [argp_usage], [argp_ldflags="-largp"])
+AC_CHECK_LIB([argp], [argp_parse], [argp_ldflags="-largp"])
 ], [AC_MSG_ERROR([Could not find argp])])
 AC_SUBST(argp_ldflags)
 
